Makefile

```
@@ -350,3 +350,17 @@ release-changelog: $(wildcard changelog/*.txt)
 	@:$(if $(LAST_RELEASE),,$(error please set the LAST_RELEASE environment variable for changelog generation))
 	@:$(if $(THIS_RELEASE),,$(error please set the THIS_RELEASE environment variable for changelog generation))
 	changelog-build -changelog-template changelog/changelog.tmpl -entries-dir changelog -git-dir . -note-template changelog/note.tmpl -last-release $(LAST_RELEASE) -this-release $(THIS_RELEASE)
+
+.PHONY: dev-rpm
+dev-rpm: dev
+	@export GORELEASER_PREVIOUS_TAG=$(shell git describe --tags $(shell git rev-list --tags --max-count=1))
+	@echo GORELEASER_PREVIOUS_TAG: $(GORELEASER_PREVIOUS_TAG)
+	@if [ -z $(GORELEASER_CURRENT_TAG) ]; then \
+		echo "ERROR: Set GORELEASER_CURRENT_TAG to a specific version"; \
+		exit 1; \
+	fi
+	@echo GORELEASER_CURRENT_TAG: $(GORELEASER_CURRENT_TAG)
+	@$(SED) 's/REPLACE_WITH_RELEASE_GOOS/linux/g' $(CURDIR)/.goreleaser-template.yaml > $(CURDIR)/.goreleaser.yaml
+	@$(SED) -i 's/^#LINUXONLY#//g' $(CURDIR)/.goreleaser.yaml
+	@export GPG_KEY_FILE=/dev/null
+	goreleaser release --clean --timeout=60m --verbose --parallelism 2 --snapshot --skip docker,sbom,sign
```

This doesn't work for me; each makefile invocation is [a separate command](https://unix.stackexchange.com/questions/656437/export-env-variable-does-not-work-from-makefile) so the export only is valid for a single line and so by the time the echo executes, it is already a new command and thus doesn't have the envvar set. 

I'd suggest:

```suggestion
	@$(SED) 's/REPLACE_WITH_RELEASE_GOOS/linux/g' $(CURDIR)/.goreleaser-template.yaml > $(CURDIR)/.goreleaser.yaml
	@$(SED) -i 's/^#LINUXONLY#//g' $(CURDIR)/.goreleaser.yaml
	@export GPG_KEY_FILE=/dev/null
	GORELEASER_PREVIOUS_TAG=$(shell git describe --tags $(shell git rev-list --tags --max-count=1)) \
		GORELEASER_CURRENT_TAG=v999.999.999 \
		GPG_KEY_FILE=/dev/null \
		goreleaser release --clean --timeout=60m --verbose --parallelism 2 --snapshot --skip docker,sbom,sign
```

(note the trailing `\` -- so that that is all one large command).

---

I chose v999.999.999 as a unique version number we'll probably never hit and thus will disambiguate the package version, but open to thoughts here.

---

The Make-native way to do this would be to declare [target-specific variables](https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html). An alternative would be to use the [`.ONESHELL`](https://www.gnu.org/software/make/manual/html_node/One-Shell.html) command for the target. One thing to be aware of though is that the default Make version shipped by MacOS is too old and doesn't support `.ONESHELL`

---

Ah yeah, that's probably cleaner. Do we expect RPMs to build on MacOS? 🤔

---

Fixed